### PR TITLE
Added more unicode characters

### PR DESCRIPTION
**Pull request by DarkGamerYT**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 
-## How to use
-Copy the unicode character and paste it into Minecraft: Bedrock Edition chats, signs, titles etc.
+### How to use
+> Copy the unicode character and paste it into Minecraft: Bedrock Edition chats, signs, titles etc.
 
-## How does it work?
-Minecraft uses resource packs to assign characters (glyphs) to different unicode values, which is how the game allows you to type in chat and write on signs etc. Because fonts are just textures in a resource pack, it also allows the game to add "emojis" which have their own unicode values assign to them just like letters.
+### How does it work?
+> Minecraft uses resource packs to assign characters (glyphs) to different unicode values, which is how the game allows you to type in chat and write on signs etc. Because fonts are just textures in a resource pack, it also allows the game to add "emojis" which have their own unicode values assign to them just like letters.
```
Comment (inxomnyaa) [inline README.md:8]: Really unrelated changes here

File: README.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
-# bedrock-unicode-characters
-Minecraft:Bedrock Edition Unicode characters
+# Bedrock Unicode Characters
+**Minecraft: Bedrock Edition** Unicode characters
 
-## How to use
-Copy the unicode character and paste it into Minecraft: Bedrock Edition chats, signs, titles etc.
+### How to use
+> Copy the unicode character and paste it into Minecraft: Bedrock Edition chats, signs, titles etc.
```
Comment (inxomnyaa) [inline README.md:5]: I don't even see a reason to use a comment block here at all

**Review by inxomnyaa — CHANGES_REQUESTED** [review]
Unrelated changes to the readme & style. I suggest separating those out into their own commit

File: README.md (hunk 2 above)
Comment (DarkGamerYT) [reply README.md:5]: I've used them a lot in my private repos so I'm kinda used to it but I can remove them!

**Review by inxomnyaa — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -78,9 +78,9 @@ Minecraft uses resource packs to assign characters (glyphs) to different unicode
 |        Middle Mouse Button       |   Windows   |          |  U+E062 |         ![](./images/middle-mouse.png)         |
 |               Mouse              |   Windows   |          |  U+E063 |             ![](./images/mouse.png)            |
 |         Forward Arrow New        |    Mobile   |          |  U+E065 |        ![](./images/forward-arrow-new.png)     |
-|          Right Arrow New         |    Mobile   |          |  U+E066 |         ![](./images/right-arrow-new.png)      |
+|          Left Arrow New          |    Mobile   |          |  U+E066 |         ![](./images/left-arrow-new.png)       |
```
Comment (Aericio) [inline README.md:81]: Were left/right in the wrong positions before? Please confirm that the unicode is correct.

File: README.md [hunk 4]
```diff
@@ -78,9 +78,9 @@ Minecraft uses resource packs to assign characters (glyphs) to different unicode
...
 |           Down Arrow New         |    Mobile   |          |  U+E067 |          ![](./images/down-arrow-new.png)      |
-|           Left Arrow New         |    Mobile   |          |  U+E068 |          ![](./images/left-arrow-new.png)      |
+|           Right Arrow New        |    Mobile   |          |  U+E068 |          ![](./images/right-arrow-new.png)     |
```
Comment (Aericio) [inline README.md:83]: Please confirm that the unicode is correct.

**Comment by Aericio** [comment]
#11 ah, I missed this PR.
